README.md

```diff
@@ -11,6 +11,9 @@ dp-frontend-renderer
 | ENABLE_COOKIES_CONTROL        | false                 | Enabled user cookie preferences
 | PATTERN_LIBRARY_ASSETS_PATH   |                       | Pattern library location
 | SUPPORTED_LANGUAGES           | [2]string{"en", "cy"} | languages supported
+| GRACEFUL_SHUTDOWN_TIMEOUT     | 5s                    | The graceful shutdown timeout
+|	HEALTHCHECK_INTERVAL          | 30s                   | Time between calls to healthchecks
```

Comment:
Some alignment issues

Reply:
it's strange - I don't see it in my vscode:
![image](https://user-images.githubusercontent.com/24572182/78661492-c740ec80-78c6-11ea-9bfd-c3b9b8f6eb3a.png)

Reply:
If you delete the whitespace inbetween | and Health<...> and then add space does git diff notice the difference?